views/aboutus/index.ejs

```diff
@@ -11,7 +11,7 @@
       <div class="row col-12"><%- include('../partials/navbar'); %></div>
 
       <div class="row col-12">
-        <div class="w-50 m-auto mt-5 mb-5" style="border-radius: 15px">
```

Comment:
I don't oppose changing the width, but why were the margins removed? They were there to balance the amount of blank space above and below the content:
![image](https://user-images.githubusercontent.com/25871926/203649523-c3e6c0e7-d8a0-4377-aac5-3c1118432779.png)
![image](https://user-images.githubusercontent.com/25871926/203649539-ca6d01fe-8382-4bfc-8d47-4f1e3e40917c.png)

Reply:
Oh okay, I'll put them back. I don't remember why I removed them.

Reply:
No worries, I didn't mind them being gone but I wanted to know how it was better than what was there originally. I like the way the text wraps much more now though.